test/csuite/tiered_abort/smoke.sh

```diff
@@ -18,7 +28,7 @@ else
     top_srcdir=${top_srcdir:-../..}
     test_bin=$top_builddir/test/csuite/test_tiered_abort
 fi
-$TEST_WRAPPER $test_bin -t 10 -T 5
-$TEST_WRAPPER $test_bin -m -t 10 -T 5
-$TEST_WRAPPER $test_bin -C -t 10 -T 5
-$TEST_WRAPPER $test_bin -C -m -t 10 -T 5
+$TEST_WRAPPER $test_bin $builddir_arg -t 10 -T 5
+$TEST_WRAPPER $test_bin $builddir_arg -m -t 10 -T 5
```

Comment:
good find! I'm interested on the original purpose of `-m` or `-C` was suppose to do? And if it is still relevant?

Reply:
This is really a question for @sueloverso .  Though I think if you leave smoke.sh to exit 0, we'll sort out what arguments make sense when we get tiered_abort running.

Reply:
@jiechenbo since `tiered_abort` started from a copy of `timestamp_abort` that is why `-m` and `-C` exist in the `smoke.sh` script. Since the script was short circuited to immediately exit, I did not do any other changes to it. Once the test is working well I am sure there will be a ticket to create a real `smoke.sh` set of tests and then a considered set of tests should be added. Adding your `$builddir_arg` is fine since it will be needed to make things run. If you want to clean up `-m` and `-C` I have no objection (then there would be some duplicate lines probably too) so go for it!

Reply:
I have removed those tests with`-m` and `-C`. Let me know if I should just comment them out instead.